conf/etc/v2/iem_ha.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Please add a comment here or in code mentioning the full IEC code. This will help in referencing and in mapping the IEC with iec_mapping file in SSPL

Reply:
I think its already there is LLD in a very proper fashion.

Reply:
In later stages of development we will not be referring to LLD much. 
Sometime down the line if any changes are need in IEC values etc, it will be easy for somenone who is not aware of the code; to have the full IEC code mentioned in the repository
@indrajitzagade , do you think it will be helpful ?

Reply:
I am doing it in the code.

Reply:
I think yes.